codec-http2/src/test/java/io/netty/handler/codec/http2/Http2CodecTest.java

```diff
@@ -110,10 +111,10 @@ public void createOutboundStream() {
         Http2Headers headers = new DefaultHttp2Headers();
         childChannel.write(new DefaultHttp2HeadersFrame(headers));
         ByteBuf data = Unpooled.buffer(100).writeZero(100);
-        childChannel.writeAndFlush(new DefaultHttp2DataFrame(data));
+        childChannel.writeAndFlush(ReferenceCountUtil.releaseLater(new DefaultHttp2DataFrame(data)));
```

Comment:
The messages are put in the outbound queue in the EmbeddedChannel and not released so they can be read from the outbound queue. This behavior differs from "real channels" in that messages are released once they are written/flushed. A more typical way to validate behavior using the EmbeddedChannel is to read from the EmbeddedChannel's outbound queue, and release messages as appropriate.